shard-operator/src/main/resources/templates/observability-secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: smartevents-observability-configuration
```

Comment:
@manstis I would make the `name` and `namespace` templatable as well and read the values from `application.properties`. It means we can (if we need to) simply update the configuration of the data plane to have the secret created with another name or in another namespace.

Reply:
Done.